src/components/shared/Select/index.tsx

```diff
@@ -166,6 +167,19 @@ const getThemeProp = ({ theme, comp, prop }: ThemePropParams): string => {
   return themeStylePropCollection[theme][comp][prop];
 };
 
+const SelectContainer = styled.View`
+  z-index: 1;
+  margin-bottom: 20px;
+`;
```

Comment:
I didn't understand why you used margin and z-index. Can you tell me why?

Reply:
Thanks a lot for your comment!
I added the code above to do some styling of the 4 Select components on the simulator preview. But now I noticed that the code is not on the right place.
I've moved those style code to 'Select.storieis.tsx' !
I updated the screenshot on the top of this page, so you can